giatrosbook.json

```diff
@@ -4,43 +4,43 @@
     "phone" : "87438807",
     "email" : "alexyeoh@example.com",
     "address" : "Blk 30 Geylang Street 29, #06-40",
-    "allergied" : [ "amoxicillin" ]
+    "allergies" : [ "ono", "rando" ],
```

Comment:
Try not to change the sample data? Or at least just add on to the existing one instead of overwriting them